src/components/FormFields/DataIntegrityChecksField.js

```diff
@@ -0,0 +1,158 @@
+import React, { useCallback, useState } from 'react'
+import { PropTypes } from '@dhis2/prop-types'
+import i18n from '@dhis2/d2-i18n'
+import {
+    FieldGroup,
+    Radio,
+    Transfer,
+    TransferOption,
+    ReactFinalForm,
+    InputFieldFF,
+    Help,
+} from '@dhis2/ui'
+import cx from 'classnames'
+import { hooks } from '../Store'
+import {
+    getCheckName,
+    severityMap,
+} from '../../services/server-translations/dataIntegrityChecks'
+import styles from './DataIntegrityChecksField.module.css'
+
+const { Field, useField } = ReactFinalForm
+
+const VALIDATOR = value =>
+    value && value.length < 1
+        ? i18n.t('Please select checks to run.')
+        : undefined
+
+const DataIntegrityChecksField = ({ label, name }) => {
+    const options = hooks.useParameterOptions('dataIntegrityChecks')
+    const {
+        input: { value, onChange },
+    } = useField(name)
+
+    const hasValue = !!value && value.length > 0
+    const [runSelected, setRunSelected] = useState(hasValue)
+
+    const translatedOptions = options
+        .map(option => ({
+            ...option,
+            value: option.name,
+            label: getCheckName(option.name),
+            severity: severityMap[option.severity],
+        }))
+        .sort((a, b) => a.label.localeCompare(b.label))
+
+    const toggle = ({ value }) => {
+        const checked = value === 'true'
+
+        if (!checked) {
+            // clear checks when "Run all" is selected
+            onChange([])
+        }
+        setRunSelected(checked)
+    }
+
+    return (
+        <FieldGroup label={i18n.t('Checks to run')}>
+            <Radio
+                name={'checksToRun'}
+                value={String(false)}
```

Comment:
Why not just write out "false"? Maybe I'm missing something?

Reply:
True, I tried a few different approaches here, since at first I wasn't aware that value had to be string. Can of course just be "false".